### ISSUE TITLE
Using dist/react-geosuggest.js with global React fails.

### ISSUE THREAD

**Issue by foot** [ubilabs/react-geosuggest#37]
E.g. 

```
bower install react
bower install react-geosuggest
```

And a basic template 

```
<!DOCTYPE HTML>
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8">

	<title>Index</title>
	<script src="bower_components/react/react.js" type="text/javascript" charset="utf-8"></script>
	<script src="bower_components/react-geosuggest/dist/react-geosuggest.js" type="text/javascript" charset="utf-8"></script>
</head>

</html>
```

I'm getting a.
```
Uncaught Error: Cannot find module 'react'
```

The README mentions that this usage should be possible.

`window.React` is exposed by react.js, but the `require`s in react-geosuggest.js can't find it, I might be missing a config option! I'm not sure if you can ask that webpack/browserify wrapper to do something to link those up.

Cheers!

We're looking into using the component in a ClojureScript/Reagent page which uses quite a different build system. It would be neat to avoid integrating a webpack/browserify step into the build for now but it might come to that anyway!

**Comment by foot** [comment]
Building a requirable react seems to work. But I'm not sure if this is the-way-to-do-it?

```browserify -r react > react-require.js```

And an updated template:

```
<!DOCTYPE HTML>
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8">

	<title>Index</title>
	<script src="https://maps.googleapis.com/maps/api/js?libraries=places"></script>
	<script src="react-require.js" type="text/javascript" charset="utf-8"></script>
	<script src="bower_components/react-geosuggest/dist/react-geosuggest.js" type="text/javascript" charset="utf-8"></script>
	<link rel="stylesheet" href="style.css" type="text/css" media="screen" charset="utf-8">
</head>

	<body>
	<div id="app">
	</div>
		
	<script type="text/javascript" charset="utf-8">
		React = require('react');
		var GeosuggestFactory = React.createFactory(Geosuggest);
		React.render(GeosuggestFactory(), document.getElementById('app'));
	</script>
	</body>
</html>

```

**Comment by foot** [comment]
This might be the go:

https://github.com/rluba/browserify-global-shim

Replaces `require('react')`s with `(window.React)`s.

**Comment by ro-ka** [comment]
Hi! Hm, will try both and see which works. Should be one file without requires at all. Thanks for reporting!

**Comment by ro-ka** [comment]
I tried the `browserify-global-shim` and `browserify-shim` but it didn’t change the generated file… Do you have any other ideas or why it didn’t work?

I used this in the `package.json`:

```json
  "browserify-shim": {
    "React": "global:react"
  },
``

**Comment by foot** [comment]
Created a PR w/ the code i used! #45

**Comment by ro-ka** [comment]
Great! Thanks a lot! Wondering why it didn’t work for me. Will release soon, finishing something more.

**Comment by ro-ka** [comment]
Released as `1.9.1`.
